scripts/setup-tools.sh

```diff
@@ -23,7 +23,7 @@ rm -fr external-tools/buildtools
 
 mkdir -p external-tools/buildtools
 echo Downloading buildtools
-curl -L -sS https://github.com/bazelbuild/buildtools/archive/6f2bf0da7e7b7c9dadd652ec7698eabb542bd0d7.tar.gz | tar zxf - --strip 1 -C external-tools/buildtools
+curl -L -sS https://github.com/bazelbuild/buildtools/archive/0.26.0.tar.gz | tar zxf - --strip 1 -C external-tools/buildtools
```

Comment:
I this a typo? Or is `0.26.0` intentional here?

Reply:
The latest release is `0.26.0`.
https://github.com/bazelbuild/buildtools/releases